doc/source/reference/routines.array-creation.rst

```diff
@@ -85,6 +85,7 @@ Numerical ranges
    meshgrid
    mgrid
    ogrid
+   lib.index_tricks.nd_grid
```

Comment:
ndgrid is an implementation for ogrid and mgrid and I would not consider it public API.
```suggestion
```